src/library_exceptions.js

```diff
@@ -417,11 +417,18 @@ var LibraryExceptions = {
     return Module['asm']['__cpp_exception'];
   },
 
-  // Given an WebAssembly.Exception object, returns the actual user-thrown
-  // C++ object address in the Wasm memory.
+  // Throw a WebAssembly.Exception object with the C++ tag. If traceStack is
+  // true, includes the stack traces within the exception object.
   // WebAssembly.Exception is a JS object representing a Wasm exception,
   // provided by Wasm JS API:
   // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WebAssembly/Exception
+  __throwCppWebAssemblyException__deps: ['$getCppExceptionTag'],
+  __throwCppWebAssemblyException: function(ex, traceStack) {
+    throw new WebAssembly.Exception(getCppExceptionTag(), [ex], {traceStack: traceStack});
```

Comment:
I wonder, is it possible to trim the stacktrace so it doesn't include the internals. e.g.:

```
    at ___throwCppWebAssemblyException (test.js:1139:13)
    at __cxa_throw (wasm://wasm/009a7c9a:wasm-function[1551]:0x24367)
```

?

Reply:
Done